pkg/test/istio.io/README.md

```diff
@@ -1,13 +1,42 @@
-# Istio.io Example Tests
+# Istio.io preliminary test cases
+
+Welcome to the development of the istio.io preliminary test cases!
+
+This work is experimental. This work is also under construction and may not work
+for you. The Istio developers have done some work to document known problems or
+workarounds. We expect this work will take several releases to complete.
 
 ## Overview
-This folder contains tests for the different examples on Istio.io.
+This folder contains tests for the different examples on [istio.io](https://istio.io).
+The goal of this work is to ensure [istio.io](https://istio.io) is tested according
+to the code in the [Istio repository](https://github.com/istio).
```

Comment:
The repository is `https://github.com/istio/istio`. `https://github.com/istio` is the organization.